docs/basic-strict.md

```diff
@@ -0,0 +1,44 @@
+# Basic Strict
+
+With a configured bootstrap, we use notes on the properties to make your code work strictly.
```

Comment:
s/`notes`/`annotations/docblocks`
s/`to make your code work strictly`/`to make your app operate in strict mode`